app/assets/stylesheets/calagator/index.scss

```diff
@@ -5,6 +5,7 @@
  * Then come the external dependencies:
  *
  *= require jquery.timepicker
+ *= require jquery-ui/theme
```

Comment:
Interesting, looks like bourbon makes mixins available to the asset pipeline by munging the `SASS_PATH` environment variable: 
https://github.com/thoughtbot/bourbon/blob/88fcaf30ff/lib/bourbon.rb#L26